src/assets/src/utils.ts

```diff
@@ -11,15 +11,6 @@ export const redirectToLogin = (loginUrl: string) => {
     location.href = loginUrl + '?next=' + location.pathname;
 }
 
-export const redirectToSearch = (term: string) => {
-    ReactGA.event({
-        category: "Navigation",
-        action: "Redirected to Search",
-        nonInteraction: true,
-    });
-    location.href = `/search/${term}/?redirected=true`;
-}
```

Comment:
Is this code just removing Google Analytics event?

Reply:
shouldn't this event be tracked? Since a user did a search and got redirect. or GA event is not tracked at all with this code?